MIGRATING.md

```diff
@@ -0,0 +1,153 @@
+# Migrate from the SignalFx Tracing Library for NodeJS
+
+The Splunk Distribution of OpenTelemetry for NodeJS replaces the SignalFx Tracing
+Library for NodeJS. If you’re using the SignalFx Tracing Library, migrate to
+the Splunk Distribution of OpenTelemetry for NodeJS to use OpenTelemetry
+instrumentation to send traces to Splunk APM. The Splunk Distribution of
+OpenTelemetry for NodeJS uses OpenTelemetry to instrument applications, which is
+an open-source API to gather telemetry data, and has a smaller footprint.
+
+Because the SignalFx Tracing Library for NodeJS uses OpenTracing and the Splunk Distribution
+of OpenTelemetry for NodeJS uses OpenTelemetry, the semantic
+conventions for span tag names change when you migrate. For more information,
+see [Migrate from OpenTracing to OpenTelemetry](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html#apm-opentelemetry-migration).
+
+## Getting help
+
+If you experience any issues following the guide below, or something is unclear, or missing, please don't hesitate to
+open an issue in Github. Any and all ideas for improvements are also welcome.
+
+<a name="known-limitations"></a>
+## Known limitations as compared to SignalFx Tracing Library
+
+- Lowest supported version of NodeJS is `v8.5`, [see more information here](https://github.com/open-telemetry/opentelemetry-js#node-support)
+- No auto-instrumentation for:
+  - `AdonisJS`
+  - `amqp10`
+  - `mongodb-core` ([because it's deprecated](https://github.com/mongodb-js/mongodb-core))
+  - `sails`
+- Limited instrumentation for:
+  - `nest` - only manual insturmentation helpers, provided by community
+- other notes on instrumentation:
+  - `express` instrumentation requires `http`/`https` instrumentation
+  - `bluebird` - context propagation only (via `async_hooks`)
+  - `q` - context propagation only (via `async_hooks`)
+  - `when` - context propagation only (via `async_hooks`)
+  - `socket.io` - provided by community (<https://github.com/aspecto-io/opentelemetry-ext-js/tree/master/packages/instrumentation-socket.io>)
+
+## Changes to defaults
+
+- Default flush interval, which defines how frequently captured telemetry data is sent to the backend, is now 30s instead of 2s
+
+## Requirements
+
+This Splunk Distribution of OpenTelemetry requires Node.js 8.5 or later.
```

Comment:
That's also stated under limitations. I'd leave one or the other.

Reply:
I'll simplify this.